interop-tests/src/test/scala/akka/grpc/scaladsl/tools/MutableServiceDiscovery.scala

```diff
@@ -27,7 +27,7 @@ final class MutableServiceDiscovery(targets: List[InetSocketAddress]) extends Se
     services = Future.successful(
       Resolved(
         "greeter",
-        targets.map(target => ResolvedTarget(target.getHostString, Some(target.getPort), Some(target.getAddress)))))
+        targets.map(target => ResolvedTarget(target.getHostString, Some(target.getPort), Option(target.getAddress)))))
```

Comment:
I think this was wrong, `Some(null)` is never a good thing. `null` is valid in InetSocketAddress. Now it will try to resolve the host, which I think was the intention with `"example.invalid"`, but that doesn't work with netty backend. Created issue #1857 for that